.appveyor.yml

```diff
@@ -71,26 +71,27 @@ platform: x86
 before_build:
   - ps: $env:Path = "$env:QT;$env:COMPILER;$env:Path"
   - sh: |
-      brew install qt jack p7zip
+      brew install qt pkg-config jack p7zip
       export PATH="/usr/local/opt/qt/bin:$PATH"
       export JACKLIB="/usr/local/opt/jack/lib"
       export JACKINC="/usr/local/opt/jack/include"
-  - cd BambooTracker
+      export PKG_CONFIG_PATH="/usr/local/opt/jack/lib/pkgconfig"${PKG_CONFIG_PATH:+':'}$PKG_CONFIG_PATH
 
 # to run your custom scripts instead of automatic MSBuild
 build_script:
   - ps: |
-      qmake.exe BambooTracker.pro PREFIX=..\target CONFIG+=debug CONFIG+=console CONFIG+=nostrip
+      qmake.exe Project.pro PREFIX=..\target CONFIG+=debug CONFIG+=console CONFIG+=nostrip
```

Comment:
Should we add `CONFIG-=release` to explicitly indicate that it is a debug?

Reply:
Just to be safe I added, that but it was most likely `PREFIX=..\target` now being wrong since we now have a top-level project file instead of `cd`ing into the Bambootracker directory